indico/util/mdx_latex.py

```diff
@@ -253,33 +268,37 @@ def tolatex(self, ournode):
         elif ournode.tag == 'q':
             buffer += "`%s'" % subcontent.strip()
         elif ournode.tag == 'p':
-            buffer += '\n%s\n' % subcontent.strip()
+            # do not escape if it is latex already
+            if contains_supported_latex_symbols(subcontent):
```

Comment:
Yes, when the content of an abstract contains LaTeX code, then without this check the PDF that gets generated looks like this:
![image](https://cloud.githubusercontent.com/assets/1579899/21887403/0a479790-d8c0-11e6-8104-222b4ec188ee.png)
whereas with the above fix it works fine:
![image](https://cloud.githubusercontent.com/assets/1579899/21887440/2f43b718-d8c0-11e6-8d97-67f64c0e710b.png)